Mathlib/Data/RBTree/Init.lean

```diff
@@ -0,0 +1,78 @@
+/-
+Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Leonardo de Moura
+
+! This file was ported from Lean 3 source module data.rbtree.init
+! leanprover-community/mathlib commit fcc158e986d4896605e97fb3ad17d5cfed49a242
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+
+import Mathlib.Init.Align
+import Std.Data.RBMap.Basic
+
+/-!
+
+# Align statements for RBTree
+
+-- This file used to be a part of `prelude`
+
+-- Porting note: this file has essentially been ported to `Std`.
+-- See `Std.Data.RBMap.Basic`
+
+-/
+
+#align rbnode Std.RBNode
+#align rbnode.color Std.RBColor
+#noalign rbnode.color.decidable_eq
+#align rbnode.depth Std.RBNode.depth
+#align rbnode.min Std.RBNode.min
+#align rbnode.max Std.RBNode.max
+#align rbnode.fold Std.RBNode.fold
+
+#noalign rbnode.rev_fold
+
+#align rbnode.balance1 Std.RBNode.balance1
+
+#noalign rbnode.balance1_node
```

Comment:
Are these things `noalign` for a reason? We don't want to port them?

Reply:
cc @digama0

Reply:
These align statements are for documentation purposes only, they are significantly different in the details. I'm not sure going over all the individual technical lemmas will be that helpful. From a cursory look it seems that `rbnode.balance1_node` is more similar to `Std.RBNode.balance1` than `rbnode.balance1`, so you could align both of them to `Std.RBNode.balance1`? :shrug: